pandas/_libs/groupby.pyx

```diff
@@ -443,26 +443,34 @@ def group_any_all(int8_t[::1] out,
     else:
         raise ValueError("'bool_func' must be either 'any' or 'all'!")
 
+    if mask is not None and mask.shape[1] != 1:
```

Comment:
Can mask be `None` here?

Reply:
yes

Reply:
i was wrong on this, will add suggested `not None` and `const`s in a separate PR